packages/grafana-data/src/types/datasource.ts

```diff
@@ -582,10 +582,29 @@ export interface QueryFix {
 
 export type QueryFixType = 'ADD_FILTER' | 'ADD_FILTER_OUT' | 'ADD_STRING_FILTER' | 'ADD_STRING_FILTER_OUT';
 export interface QueryFixAction {
-  type: QueryFixType | string;
   query?: string;
   preventSubmit?: boolean;
+  /**
+   * The type of action to perform. Will be passed to the data source to handle.
+   *
+   * @type {(QueryFixType | string)}
+   * @memberof QueryFixAction
+   */
```

Comment:
Maybe remove @memberof? Or is it useful for stuff like Intellij? It's not used anywhere in the repo.

Reply:
Removed both unnecessary props in 84d0b11e8408d76bc0821043541d96fa2c3e8dde